website/docs/r/nsxt_network_segment_profile.html.markdown

```diff
@@ -0,0 +1,152 @@
+---
+
+# vcd\_nsxt\_network\_segment\_profile
+
+Provides a resource to configure Segment Profiles for NSX-T Org VDC networks.
+
+Supported in provider *v3.11+* and VCD 10.4.0+ with NSX-T.
+
+## Example Usage (Segment Profile Template assignment to Org VDC Network)
+
+```hcl
+data "vcd_nsxt_segment_profile_template" "complete" {
+  name = "complete-profile"
+}
+
+data "vcd_nsxt_edgegateway" "existing" {
+  org  = "my-org"
+  name = "my-gw"
+}
+
+resource "vcd_network_routed_v2" "net1" {
+  org  = "my-org"
+  name = "routed-net"
+
+  edge_gateway_id = data.vcd_nsxt_edgegateway.existing.id
+
+  gateway       = "1.1.1.1"
+  prefix_length = 24
+
+  static_ip_pool {
+    start_address = "1.1.1.10"
+    end_address   = "1.1.1.20"
+  }
+}
+
+resource "vcd_nsxt_network_segment_profile" "custom-prof" {
+  org            = "my-org"
+  org_network_id = vcd_network_routed_v2.net1.id
+
+  segment_profile_template_id = data.vcd_nsxt_segment_profile_template.complete.id
+}
+```
+
+## Example Usage (Custom Segment Profile assignment to Org VDC Network)
+
+```hcl
+data "vcd_nsxt_manager" "nsxt" {
+  name = "nsxManager1"
+}
+
+data "vcd_nsxt_segment_ip_discovery_profile" "first" {
+  name            = "ip-discovery-profile-0"
+  nsxt_manager_id = data.vcd_nsxt_manager.nsxt.id
+}
+
+data "vcd_nsxt_segment_mac_discovery_profile" "first" {
+  name            = "mac-discovery-profile-0"
+  nsxt_manager_id = data.vcd_nsxt_manager.nsxt.id
+}
+
+data "vcd_nsxt_segment_spoof_guard_profile" "first" {
+  name            = "spoof-guard-profile-0"
+  nsxt_manager_id = data.vcd_nsxt_manager.nsxt.id
+}
+
+data "vcd_nsxt_segment_qos_profile" "first" {
+  name            = "qos-profile-0"
+  nsxt_manager_id = data.vcd_nsxt_manager.nsxt.id
+}
+
+data "vcd_nsxt_segment_security_profile" "first" {
+  name            = "segment-security-profile-0"
+  nsxt_manager_id = data.vcd_nsxt_manager.nsxt.id
+}
+
+data "vcd_nsxt_edgegateway" "existing" {
+  org  = "my-org"
+  name = "nsxt-gw-v40"
+}
+
+resource "vcd_network_routed_v2" "net1" {
+  org  = "my-org"
+  name = "routed-net"
+
+  edge_gateway_id = data.vcd_nsxt_edgegateway.existing.id
+
+  gateway       = "1.1.1.1"
+  prefix_length = 24
+
+  static_ip_pool {
+    start_address = "1.1.1.10"
+    end_address   = "1.1.1.20"
+  }
+}
+
+resource "vcd_nsxt_network_segment_profile" "custom-prof" {
+  org            = "my-org"
+  org_network_id = vcd_network_routed_v2.net1.id
+
+  ip_discovery_profile_id     = data.vcd_nsxt_segment_ip_discovery_profile.first.id
+  mac_discovery_profile_id    = data.vcd_nsxt_segment_mac_discovery_profile.first.id
+  spoof_guard_profile_id      = data.vcd_nsxt_segment_spoof_guard_profile.first.id
+  qos_profile_id              = data.vcd_nsxt_segment_qos_profile.first.id
+  segment_security_profile_id = data.vcd_nsxt_segment_security_profile.first.id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional, but required if not set at provider level) Org name 
```

Comment:
(Same as in above comment). Let's keep this part consistent with other pages, and let's keep the part in parenthesis canonical - without comments.

Reply:
Fixed.